### DISCUSSION TITLE
Display overall "best sectors" of the session

### DISCUSSION THREAD

**Discussion by Walker1390** [f1multiviewer/issue-tracker#128] [Feature requests] [answered]
As [discussed in discord](https://discord.com/channels/982375740764983366/984792219573694465/1000421785864503347). Display the best sector so when watching a drivers lap we have something to compare the sector times to. Also the option of getting a drivers best sector times when hovering over them or something like that.

**Comment by f1multiviewer** [comment, marked as answer]
Done!
